### PR TITLE
French translation of README

### PR DESCRIPTION
**Pull request by BrightDV**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

Add translation of the French README file.

#### APK testing 
The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <p align="center"><a href="https://newpipe.net">Website</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Press</a></p>
 <hr>
 
-*Read this in other languages: [English](README.md), [Español](README.es.md), [한국어](README.ko.md), [Soomaali](README.so.md), [Português Brasil](README.pt_BR.md), [日本語](README.ja.md), [Română](README.ro.md), [Türkçe](README.tr.md), [正體中文](README.zh_TW.md).*
+*Read this in other languages: [English](README.md), [Español](README.es.md), [Français](README.fr.md), [한국어](README.ko.md), [Soomaali](README.so.md), [Português Brasil](README.pt_BR.md), [日本語](README.ja.md), [Română](README.ro.md), [Türkçe](README.tr.md), [正體中文](README.zh_TW.md).*
```
Comment (mhmdanas) [inline README.md:20]: You have to do this for all the READMEs, including the French one.

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BrightDV) [reply README.md:20]: Ok

File: README.fr.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+NewPipe supporte plusieurs services. Notre [documentation](https://teamnewpipe.github.io/documentation/) donne plus d'informations concernant l'ajout d'un nouveau service à l'application et à l'extracteur. Merci de nous contacter si vous avez l'intention d'en rajouter un nouveau. Pour l'instant, les services supportés sont:
+
+* YouTube
+* SoundCloud \[bêta\]
+* media.ccc.de \[bêta\]
+* Les instances PeerTube \[bêta\]
+* Bandcamp \[bêta\]
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
```
Comment (mhmdanas) [inline README.fr.md:88]: This is not needed here (see https://github.com/TeamNewPipe/NewPipe/pull/7694#discussion_r791011177):
```suggestion
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: README.fr.md (hunk 2 above)
Comment (BrightDV) [reply README.fr.md]: OK, fixed !

File: README.fr.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+* Rechercher dans l'historique
+* Rechercher/Voir des listes de lecture
+* Mettre et voir les listes de lecture en file d'attente
+* Mettre des vidéos en queue
+* Listes de lecture locales
+* Sous-titres
+* Voir des vidéos en direct
+* Afficher les commentaires
+
+### Supported Services
```
Comment (yannjor) [inline README.fr.md:77]: Should this header be translated?

Comment (mhmdanas) [reply README.fr.md:77]: Yeah, all headers should be translated. Don't forget to update the links at the top of the README.

File: README.fr.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    <td><a href="https://liberapay.com/TeamNewPipe/donate"><img src="assets/liberapay_donate_button.svg" alt="Donner via Liberapay" height="35px"></a></td>
+  </tr>
+  <tr>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="assets/bountysource_qr_code.png" alt="Visit NewPipe at bountysource.com" width="100px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="Check out how many bounties you can earn."></a></td>
+  </tr>
+</table>
+
+## Police de Confidentialité
```
[the remark this answers is not in the document]
Comment (BrightDV) [reply README.fr.md:133]: Yes ! Thank you for your review !

**Comment by BrightDV** [comment]
Is it ok as of now?

**Comment by mhmdanas** [comment]
I can't find any general issues, but I don't know French, so I can't review the text itself.

In any case, please rebase your PR.

File: README.fr.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+* Pas de compte nécessaire
+* Afficher les informations générales à propos de vidéos
+* Voir des vidéos YouTube
+* Écouter des vidéos YouTube
+* Mode flottant (lecteur flottant)
+* Choisir le lecteur avec lequel voir la vidéo
+* Télécharger des vidéos
+* Télécharger seulement le son des vidéos
+* Ouvrir une vidéo avec Kodi
+* Voir les vidéos suivantes/relatées
```
Comment (pihug12) [inline README.fr.md:58]: "relatées" me parait bizarre. "liées" ? ou "apparentées" ? ou "suggérées" ?

File: README.fr.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+* Rechercher sur YouTube dans un langage specifique
```
Comment (pihug12) [inline README.fr.md:59]: dans une langue spécifique ?

File: README.fr.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+* Rechercher des chaînes
+* Regarder les vidéos d'une chaîne
+* Supporte Orbot/Tor (pas encore directement)
+* Supporte les résolutions 1080p/2K/4K
+* Voir l'historique
+* S'abonner à des chaînes
+* Rechercher dans l'historique
+* Rechercher/Voir des listes de lecture
+* Mettre et voir les listes de lecture en file d'attente
+* Mettre des vidéos en queue
```
Comment (pihug12) [inline README.fr.md:71]:
```suggestion
* Mettre des vidéos en file d'attente
```

File: README.fr.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+* media.ccc.de \[bêta\]
+* Les instances PeerTube \[bêta\]
+* Bandcamp \[bêta\]
+
+## Installation et mise à jour
+Vous pouvez installer NewPipe en utilisant l'une de ces méthodes :
+ 1. Ajoutez notre repository à F-Droid et installez l'application de là. Les instructions sont ici : https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
+ 2. Téléchargez-la depuis [les publications d'applications sur GitHub](https://github.com/TeamNewPipe/NewPipe/releases) et installez-là.
+ 3. Mettre à jour via F-Droid. C'est la méthode la plus lente pour avoir les mises à jour car F-Droid doit reconnaître les changements, construire l'application elle-même, la signer et ensuite la publier auprès des utilisateurs.
+ 4. Construire une verision de débogage soi-même. C'est la méthode la plus rapide pour avoir les nouvelles fonctionnalités sur votre appareil, mais c'est plus compliqué, donc nous vous recommandons d'utiliser une autre méthode.
```
Comment (pihug12) [inline README.fr.md:92]: verision --> version

File: README.fr.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+
+Nous recommandons la première méthode pour la majorité des utilisateurs. Les fichiers APKs installés en utilisant les méthodes 1 et 2 sont compatibles entre eux, mais pas avec ceux installés via la méthode 3. Cela est dû à l'utilisation de la même clé de signature (la nôtre) pour les méthodes 1 et 2, et une clé de signature différente (celle de F-Droid) pour la méthode 3. Construire une version de débogage en utilisant la méthode 4 exclue totalement la clé de signature. Les clés de signures permettent de vérifier que l'utilisateur ne se trompe pas en installant une mise à jour malveillante de l'application.
```
Comment (pihug12) [inline README.fr.md:94]: signures -> signature

File: README.fr.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+En attendant, si vous souhaitez changer de source (par exemple les fonctionnalités principales de NewPipe sont cassées et F-Droid n'a pas encore la mise à jour qui les corrigent), nous vous recommandons d'utiliser la procédure suivante :
+1. Sauvegardez vos données via Paramètres > Contenu > Exporter la base de données, ainsi, vous conservez votre historique, vos abonnements et vos listes de lecture.
+2. Désinstallez NewPipe.
+3. Téléchargez le fichier APK depuis la nouvelle source et installez-le.
+4. Importez les données de l'étape 1 via Paramètres > Contenu > Importer la base de données.
+
+## Contribution
+Si vous avez des idées, des traductions, des changements de design, des nettoyages de code, ou de grands changements de code, l'aide est toujours appréciée !
+Plus est fait, mieux c'est !
```
Comment (pihug12) [inline README.fr.md:104]: Plus il y en a, mieux c'est !

File: README.fr.md [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+
+Si vous souhaitez participer, regardez nos [notes de contribution](.github/CONTRIBUTING.md).
+
+<a href="https://hosted.weblate.org/engage/newpipe/">
+<img src="https://hosted.weblate.org/widgets/newpipe/fr/287x66-grey.png" alt="Statut de la traduction" />
+</a>
+
+## Donation
+Si vous aimez NewPipe, nous serions heureux d'une donation. Vous pouvez envoyer du Bitcoin, ou donner via Bountysource ou Liberapay. Pour plus d'informations concernant les donations pour NewPipe, merci de visiter notre [site internet](https://newpipe.net/donate).
```
Comment (pihug12) [inline README.fr.md:113]: une donation -> un don ?

Comment (BrightDV) [reply README.fr.md:113]: Oui, j'ai fait des recherches et effectivement, un don correspond mieux à la situation !
Merci pour les retours

File: README.fr.md (hunk 5 above)
Comment (BrightDV) [reply README.fr.md:58]: Effectivement... J'ai changé pour "similaires" afin d'utiliser le même terme que dans l'application

**Comment by BrightDV** [comment]
I think this pr is ready for review!

**Comment by BrightDV** [comment]
@TiA4f8R Can you please check the pr as I think it is ready?

**Comment by litetex** [comment]
Ping @TiA4f8R

**Comment by AudricV** [comment]
Closing in favor of #9296.

**Pull request closed without merging**